msteams-platform/bots/how-to/conversations/conversation-messages.md

```diff
@@ -244,7 +244,7 @@ Messages received from or sent to your bot can include different types of messag
 
 ## Notifications to your message
 
-You can also add notifications to your message using the `Notification.Alert` property. Notifications alert users about new tasks, mentions, and comments. These alerts are related to what users are working on or what they must look at by inserting a notice into their activity feed. For notifications to trigger from your bot message, set the `TeamsChannelData` objects `Notification.Alert` property to *true*. Whether or not a notification is raised depends on the individual user's Teams settings and you can't override these settings. The notification type is either a banner, or both a banner and an email.
+You can add notifications to your message using the `Notification.Alert` property. The notifications alert users about new tasks, mentions, and comments. These alerts are related to what users are working on or what they must look at by inserting a notice into their activity feed. You can use the activity feed notification APIs in Microsoft Graph to extend the functions to your apps. For notifications to trigger from your bot message, set the `TeamsChannelData` objects `Notification.Alert` property to *true*. Whether or not a notification is raised depends on the individual user's Teams settings and you can't override these settings. The notification type is either a banner, or both a banner and an email. For more information, see [how to send activity feed notifications using Graph API](/graph/teams-send-activityfeednotifications) along with the [best practices](/graph/teams-activity-feed-notifications-best-practices).
```

Comment:
"You can use the activity feed notification APIs in Microsoft Graph to extend the functions to your apps." 

This sentence reads like it is directly linked and important to the information around it, when in fact the only correlation is that the ISV is sending a notification. It should be clearer that that Bot and Graph based notifications are different and are to be used in different scenarios.

An alternative to how we discuss Graph based notifications might be to have a callout that briefly explains the difference between Bot and Graph based notifications and then directs them to the Graph docs. You might be able to summarize the most important use cases of Graph notifications as described in the "Best practices" section: https://docs.microsoft.com/en-us/graph/teams-activity-feed-notifications-best-practices#activity-feed-notifications-or-bot-framework-messages

Reply:
Updated